consensus/cryptarchia-engine/src/config.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{Epoch, Slot};
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct TimeConfig {
+    // How long a slot lasts in seconds
+    pub slot_duration: u64,
+    // Start of the first epoch, in unix timestamp second precision
+    pub chain_start_time: u64,
+}
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct Config {
+    // Depth of forks that trigger the density rule
+    pub k: u8,
+    // Number of slots after forking point to consider for the density rule
+    pub s: u8,
```

Comment:
This should be derived from `k`. It should be `s = 3 * k / f`, or `s = 3 * self.self.base_period_length()`

And the meaning behind `s` is the number of slots required to wait to have a high confidence that at least `k` blocks have been produced.